manifests/params.pp

```diff
@@ -70,6 +70,7 @@
   $enable_katello = false
   $enable_crane = false
   $enable_rpm = true
+  $enable_deb = true
```

Comment:
If we default to true we should ensure it's present in repositories. At least for Katello this is not true at the time of writing. Not sure about upstream.